code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -107,6 +108,22 @@ GLOBAL_LIST_EMPTY(PDAs)
 		return attack_self(M)
 	return
 
+/obj/item/device/pda/ui_action_click()

```

Comment:
I can't find a single other item that doesn't use ui_action_click() as an example for how to do that